### PR TITLE
Fix unicode error and remove unused import.

### PR DESCRIPTION
**Pull request by busykoala**

### CODE REVIEW

**Comment by busykoala** [comment]
@jone könntest du bitte das onegovbox hier hinzufügen?

**Comment by jone** [comment]
✅

File: ftw/oidcauth/browser/oidc_tools.py [hunk 1]
```diff
@@ -265,7 +265,7 @@ def update_login_times_and_other_member_properties(self, member):
     def setup_session(self):
         uf = api.portal.get_tool('acl_users')
         uf.updateCredentials(
-            self.request, self.request.response, self.userid, '')
+            self.request, self.request.response, safe_utf8(self.userid), '')
```
Comment (mbaechtold) [inline ftw/oidcauth/browser/oidc_tools.py:268]: Could this be tested with a userid containing an umlaut?

Comment (busykoala) [reply ftw/oidcauth/browser/oidc_tools.py]: I added a test that even helped me to improve/fix the implementation.

File: ftw/oidcauth/browser/oidc_tools.py [hunk 2]
```diff
@@ -16,6 +15,7 @@
 from jwt.exceptions import DecodeError
 from jwt.exceptions import InvalidTokenError
 from plone import api
+from plone.dexterity.utils import safe_unicode
```
Comment (mbaechtold) [inline ftw/oidcauth/browser/oidc_tools.py:18]: Is `plone.dexterity` a dependency of this package? If not, there is another implementation in `Products.CMFPlone.utils.safe_unicode`.

Comment (busykoala) [reply ftw/oidcauth/browser/oidc_tools.py:18]: It's not a direct dependency but required by:

**Plone 4.3**
- plone.app.layout
- plone.app.dexterity
- plone.app.uuid

**Plone 5.1**
- plone.app.layout
- plone.app.multilingual
- plone.app.intid
- plone.app.contenttypes
- plone.app.dexterity
- plone.app.versioningbehavior
- plone.app.uuid
- plone.app.lockingbehavior
- plone.app.event

**Pull request merged**